wdns/edns_options.c

```diff
@@ -189,6 +189,8 @@ _wdns_ednsoptdata_to_ubuf(ubuf *u, uint16_t option_code, const uint8_t *src, uin
 			break;
 		}
 		case extended_dns_error: {
+			char tmp[10];
```

Comment:
Style point -- rather than the number `10`,
```
char tmp[sizeof("65535")];
```
would clarify that this buffer is intended to hold a `uint16_t` in decimal form.